README.md

```diff
@@ -1,16 +1,36 @@
-# Azure Key Vault Provider for Secret Store CSI Driver
+# Azure Key Vault Provider for Secrets Store CSI Driver
 
 [![Build Status](https://dev.azure.com/azure/secrets-store-csi-driver-provider-azure/_apis/build/status/secrets-store-csi-driver-provider-azure-ci?branchName=master)](https://dev.azure.com/azure/secrets-store-csi-driver-provider-azure/_build/latest?definitionId=67&branchName=master)
 
-Azure Key Vault provider for [Secret Store CSI driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver) allows you to get secret contents stored in an [Azure Key Vault](https://docs.microsoft.com/en-us/azure/key-vault/general/overview) instance and use the Secrets Store CSI driver interface to mount them into Kubernetes pods.
+Azure Key Vault provider for [Secrets Store CSI driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver) allows you to get secret contents stored in an [Azure Key Vault](https://docs.microsoft.com/en-us/azure/key-vault/general/overview) instance and use the Secrets Store CSI driver interface to mount them into Kubernetes pods.
+
+## Features
+
+- Mounts secrets/keys/certs on pod start using a CSI volume
+- Supports mounting multiple secrets store objects as a single volume
+- Supports pod identity to restrict access with specific identities
+- Supports pod portability with the SecretProviderClass CRD
+- Supports windows containers (Kubernetes version v1.18+)
+- Supports sync with Kubernetes Secrets (Secrets Store CSI Driver v0.0.10+)
+- Supports multiple secrets stores providers in the same cluster.
+
+#### Table of Contents
+
+- [Demo](#demo)
+- [Usage](#usage)
+  - [Install with Helm](#install-the-secrets-store-csi-driver-and-the-azure-keyvault-provider)
+  - [Using the Azure Key Vault Provider](##using-the-azure-key-vault-provider)
+- [Azure Key Vault Provider Features](#azure-key-vault-provider-features)
+- [Troubleshooting](#troubleshooting)
+- [Contributing](#contributing)
```

Comment:
Should we also add `Testing` and `Support` to the table of contents?

Reply:
Good call.